### PR TITLE
4800 rename run-protocol package to inter-protocol

### PR DESCRIPTION
**Pull request by turadg**
part of: #4800 

## Description

Single commit for review to include in the mega https://github.com/Agoric/agoric-sdk/pull/5585

### Security Considerations

no change

### Documentation Considerations

Do we need a package redirect notice? I figure SDK is understood to be in flux, and the `agoric` command works as well.

### Testing Considerations

--

### CODE REVIEW

File: packages/cosmic-swingset/Makefile [hunk 1]
```diff
@@ -49,7 +49,7 @@ $(EVAL_CLEAN): $(EVAL_CODE)
 	./scripts/clean-core-eval.js $< >$@.t || { rm -f $@.t; exit 1; }
 	mv $@.t $@
 
-RUN_PROTO=../run-protocol
+RUN_PROTO=../inter-protocol
```
Comment (Chris-Hibbert) [inline packages/cosmic-swingset/Makefile]: Why leave RUN_PROTO behind?

File: packages/cosmic-swingset/Makefile [hunk 2]
```diff
@@ -71,8 +71,8 @@ gov-q:
 	$(AGCH) query gov proposals --output json | \
 		jq -c '.proposals[] | [.proposal_id,.voting_end_time,.status]'
 
-deploy-runStake: ../run-protocol/test/runStake/runStake-deploy.js
-	echo agoric deploy ../run-protocol/test/runStake/runStake-deploy.js
+deploy-runStake: ../inter-protocol/test/runStake/runStake-deploy.js
```
Comment (Chris-Hibbert) [inline packages/cosmic-swingset/Makefile]: Is runStake renaming a separate pass?

**Review by Chris-Hibbert — COMMENTED** [review]
search for "RUN [pP]rotocol" in .../governance and .../inter-protocol, ignoring CHANGELOG files.

File: packages/cosmic-swingset/Makefile (hunk 1 above)
Comment (turadg) [reply packages/cosmic-swingset/Makefile]: not intentional. this was a find-replace and I haven't looked over all the instances yet. I figured I would keep putting commits into #5585 but I'll get more into this one and mark it as Draft until I think it's ready to merge.

File: packages/cosmic-swingset/Makefile (hunk 2 above)
Comment (turadg) [reply packages/cosmic-swingset/Makefile]: yep, needed to get alignment on the new name

**Pull request closed without merging**